lib/src/main/java/com/auth0/android/lock/adapters/CountryAdapter.java

```diff
@@ -114,12 +112,8 @@ protected FilterResults performFiltering(CharSequence constraint) {
         @Override
         @SuppressWarnings("unchecked")
         protected void publishResults(CharSequence constraint, FilterResults results) {
-            data = (List<Country>) results.values;
-            if (results.count > 0) {
-                notifyDataSetChanged();
-            } else {
-                notifyDataSetInvalidated();
```

Comment:
docs about this method say that once called, the adapter is no longer valid. https://developer.android.com/reference/android/widget/BaseAdapter#notifyDataSetInvalidated()